packages/wrangler/src/workflows/commands/delete.ts

```diff
@@ -0,0 +1,33 @@
+import { fetchResult } from "../../cfetch";
+import { defineCommand } from "../../core";
+import { logger } from "../../logger";
+import { requireAuth } from "../../user";
+
+defineCommand({
+	command: "wrangler workflows delete",
+	metadata: {
+		description:
+			"Delete workflow - when deleting a workflow, it will also delete it's own instances",
+		owner: "Product: Workflows",
```

Comment:
I considered this in the design but I think the granularity and colocation trump the duplication and especially since it will be infrequently changed